.devcontainer/Dockerfile

```diff
@@ -320,7 +322,15 @@ RUN ldconfig -v
 
 
 ##### Install Python requirements
-RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0"
+RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0" "virtualenv==20.0.17"
+# use same virtualenv version that is installed in VM
+# TODO: install virtualenv with apt instead?
```

Comment:
Do I understand correctly, that it is preferable to install virtualenv with apt in this case? Changing it.

Reply:
You could install either way. I slightly prefer apt, however pip would also work for me.

Out of scope (of this PR), but interesting:
Is there a special necessity for that specific version of setuptools?
Otherwise, if an previous release would also work, we could also install setuptools with `apt install python3-setuptools=45.2.0-1`.
Advantage would be, that we can remove the complete pip3 install step.

Reply:
It is the version that is pinned in setup.py files. (Though in the `python.mk` file a comment says that it has to be "newer than 41.0.1" - but not sure if that is still valid.)